### PR TITLE
Added a Conditional GAN lab

### PR DESCRIPTION
**Pull request by takumiohym**
Added a new lab for conditional GAN.
Lab exercise version will follow after the review.

### CODE REVIEW

File: notebooks/image_models/solutions/conditional_gan.ipynb [hunk 1]
```diff
@@ -0,0 +1,1189 @@
+{
```
Comment (kylesteckler) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Add these imports to the cell above to have them all in one cell </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (kylesteckler) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>map -&gt; maps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (kylesteckler) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Plot generator model.
</pre><p>Should be "Plot discriminator model."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (kylesteckler) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>This header seems out of place here. Maybe move this down and replace this with a header saying "Create input data pipeline with tf.data" or something along those lines. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (kylesteckler) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>I would add a markdown note about training time here. With a T4 it took about 7 minutes to train 10 epochs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Done. Thanks!</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Makes sense. Added that line! Thanks!</p>

File: notebooks/image_models/solutions/conditional_gan.ipynb [hunk 2]
```diff
@@ -0,0 +1,1190 @@
...
```
Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Please a WARNINGS file in this directory and add an entry mentioning that the exact GPU setup this labs need to run on.</p><p>Remove the GPU test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <ul><li>This function name is confusing. It seems it crate labelled images. Change it to "create_label_embedding" or something describing more precisely what's happening.</li><li>This code would be much more readable without the <code>tf.keras.layers</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <ul><li>Same comment: The <code>tf.keras.layers</code> plus formatting make this code somehow not easy on the eye</li><li>Add a explanation paragraph above explaining what this function does and why we need it</li><li>Generally, I'd remove the function comments and instead explain what they do in a markdown cell above the function. It feels the code in this lab is unnecessary long and verbose</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <ul><li>Why is this code so verbose. It's a 3-liner?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <ul><li>Why standard in the function name? Remove?</li><li>It feels the "create_genrator_block" is a bit overkill. It makes the structure of the generator harder to parse at a glance, and defining the create_generator_block takes as long as defining it in terms of lines of code. I suggest merging this cell and the one above and making the network structure very apparent with the code formatting (maybe remove the prefix <code>tf.keras.layers</code>).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>With all the helpers function I find the overall architecture hard to follows. Is there a way to make all the code in this lab shorter and the structure more apparent? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>I find this code unnecessary long and hard to follow. I'd remove the <code>create_discriminator_block</code> helper, and have a single function. I'd move the comments in an explanatory markdown cell above, and  remove the <code>tf.keras.layers</code> prefix.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Link the previous lab in markdown</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Split this into two cells. Imports and checking GPU. Incase someone wants to debug the GPU, they don't need to import each time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>nit: ...should be a rank 3 tensor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #18.        num_pixels = image_shape[0] * image_shape[1]
</pre><p>Add a comment above explaining how this computes the num of pixels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Add some description in markdown above this cell explaining latent vectors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>This is too much code with no descriptions, I would suggest breaking this up into smaller cells and explaining what is going on in each cell. You have 3 functions, you can split it up into 3 cells and explain each function a little more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Same comment as before, this is too much code for one cell. Consider splitting this cell up.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (sanjanalreddy) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>7 minutes for how many epochs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (sanjanalreddy) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Also, what kind of a GPU

**Review by sanjanalreddy — CHANGES_REQUESTED** [review]
I feel like the lab could use a little more work in making it approachable. Please see the comments.

File: notebooks/image_models/solutions/conditional_gan.ipynb (hunk 2 above)
Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>The entire code in this cell is one class actually. I will try to reduce the size, but is it ok to put it in one cell? Or do you think it is better to apply different approach?</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Added. Thanks!</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Modified.Thanks.</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Thanks for your comment.</p><p>Since this is a single class and I believe this is the minimum setup, I added a comprehensive explanations in the markdown cell above.</p><p>Hope it works for you!</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>Modified. Thanks.</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: I will add a file after this. So far removed the file.
And can I ask you why `WARNING` file? Is it ok to put in `README.md`? @BenoitDherin

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Thanks. I modified the name and removed `tf.keras.layers` from this lab.

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: I removed `tf.keras.layers` from the lab.
And removed helper functions. Thanks!

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Removed the helper functions and merged into one functions. Thanks!

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Removed! Thanks

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Helper functions removed! Thanks

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Removed! Thanks

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Removed GPU check based on Benoit's comment. Thanks.

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Added a comment on it. Thanks

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: I think it is explained at the beginning of the lab with diagram. Does it work?

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb:1]: Removed Helper function and added some explanations. Thanks

File: notebooks/image_models/solutions/conditional_gan.ipynb [hunk 3]
```diff
@@ -0,0 +1,1139 @@
...
```
Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>This looks much clearer! Great job refactoring it. Maybe add a comment on that <code>projection_shape. is</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>* Really great refactor! Super clear now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <p>It feels still too much code at one. Here is a way to simplify:</p><p> </p><ul><li class="ql-indent-1">Define the following method as standalone functions outside the class. This will give you the opportunity to explain in markdown cells what they do and will make the overall code intake much more palatable</li><li class="ql-indent-2"> <code>generator_train_step(generator, batch_size, input_digits)</code>  </li><li class="ql-indent-2"> 	<code>discriminator_train_step(discriminator, batch_size, input_digits)</code></li><li class="ql-indent-2"> 	<code>rain_step(self, features)</code></li></ul><p> </p><ul><li class="ql-indent-1">Consider switching to a custom training loop directly instead of packaging the custom training code into a Keras model. Is there any advantage of packaging the custom training code into a Keras Model that? It feels to me that it adds complexity and remove opportunities to explain what is really happening in the training loop.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (BenoitDherin) [inline notebooks/image_models/solutions/conditional_gan.ipynb:1]: <ul><li>Some <code>tf.keras.layers</code> prefix are leftover </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb]: <p>Thank you!</p><p>I rewrote the markdown above to include <code>projection_shape</code>.</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb]: <p>Thank you!</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb]: <p>Oh thanks. I removed them.</p>

Comment (takumiohym) [reply notebooks/image_models/solutions/conditional_gan.ipynb]: <p>Thanks for your input!</p><p>I moved this subclassing model to the optional section, and added a custom training loop functions in the main section.</p><p>Please take a look at that part again.</p>

**Comment by takumiohym** [comment]
<p>Modified the line to alight other Conv2dTranspose layer formats. Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thank you!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thank you :)</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thank you :)</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>That makes sense. I added an explanation here.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by sanjanalreddy — APPROVED** [review]
Once the minor comments are addressed. Looks good to merge!

**Review by BenoitDherin — APPROVED** [review]
LGTM.

**Comment by takumiohym** [comment]
<p>Thanks for catching it! Modified.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thank you! Modified.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Actually we removed the argument names to cut shorten the code. </p><p>Since it is wouldn't be the first time to use Conv2d and Conv2dTranspose, I think it is OK. What do you think?</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Removed! Thanks</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Modified. Thanks for the suggestion.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Actually no reasoning of this number. Since default value is sufficient I will just remove the buffer size. Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sanjanalreddy** [comment]
<p>I see! Do you mean they would have seen this layer in another lab?  If so, can you just add a comment above one of the Conv2D layer to explain the arguments once so the lab can be independent of the other lab? To me, it's hard to know what these numbers are without going into the documentation.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sanjanalreddy** [comment]
<p>Just wanted to make the update the text to the one I suggested to make it more readable. Since training alternately is not common.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>I will close this so far based on our discussion. Thank you!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>I will close this so far based on our discussion. Thank you!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>I will close this so far based on our discussion. Thank you!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Thanks for the clarification. I agree and modified :)</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Added the discriminator accuracy to the code itself. Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
<p>Added a comment about it. Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by takumiohym** [comment]
@sanjanalreddy I modified the code based on our discussion! Please merge it if it looks good to you :) Thanks!
